upgrading.md

```diff
@@ -0,0 +1,202 @@
+# Upgrading (very old) WordPress
+
+Upgrading WordPress should be a simple task if its maintained and updated over time and adapted to each major version with the [requirements of that moment](https://make.wordpress.org/hosting/handbook/server-environment/).
+
+**What happens with older versions?**
+
+WordPress provides security support from WordPress 3.7 to the latest version (WordPress 6.0, __when creating this document__), each with its own database versions, PHP versions and a series of compatibilities that need to be updated over time.
+
+**What if it is not updated?**
+
+On this page there are some ideas and suggestions on how to upgrade. In many cases it will not be possible to upgrade the operating system, include new PHP versions, so will have to migrate WordPress to an updated hosting.
+
+Many assumptions will be made in this document, as each case will be different. Before doing any kind of upgrade, please make a backup of all site data: WordPress files, Database, Certificates and OS / Services configurations.
+
+IMPORTANT: This is a very manual process, not a massive one. There will be probably losses and some updates will be needed after the upgrade.
+
+[tip]In case only a major version (X.X) is indicated, use its minor (X.X.x) latest version available.[/tip]
+
+## Upgrading from WordPress 0.7 - 3.6 (by migration)
+
+Goals:
+- WordPress: upgrade to WordPress 6.0
+- PHP: upgrade to PHP 7.4
+- SQL: upgrade to MySQL 8.0 / MariaDB 10.5
+
+Losses:
+- Content: none
+- Plugins: all
+- Themes: all
+
+These are the oldest versions of WordPress and the ones that have not been supported for years. In general, have to assume some losses, although not of the contents, but probably of some functionality on themes and plugins.
+
+Considering that the goal is to keep the contents and assuming the loss of the rest of the elements, there are some steps.
+
+As with any upgrade, the first thing to do is to make a backup copy. The best way to upgrade from WP < 3.6 is to perform a content migration. 
+
+1. Create a brand-new WordPress, withoud the database.
+2. Copy the old WordPress files from the "/wp-content/uploads/" content to the new one.
+3. Create a new database with the old database information. The best way is using "mysqldump".
+4. Configure the wp-config.php with all the new data.
+5. Access the "/wp-admin/" page, and follow the upgrading process.
+
+With this way, WordPress will be able to maintain and update the contents in the database and be able to work with these contents in an updated version of WordPress.
+
+A WordPress with the default theme, and all the contents should now be available.
+
+A common problem found in these database restorations is the character encoding type. It is very likely that its not in UTF-8 (maybe in an ISO or ASCII "deprecated" format), so make sure that the character encoding is updated correctly.
```

Comment:
suggest: detail how to check the character encoding

Reply:
Recommending that we update this wording to something like: 
"Character Encoding type commonly presents technical hiccups when restoring a database. It is possible that the backup data not encoded in UTF-8 (and instead may be in an ISO or ASCII "deprecated" format). Make sure that the character encoding is updated correctly upon restoring a database! [More information on converting Character Sets in a WordPress database can be found here.](https://codex.wordpress.org/Converting_Database_Character_Sets) "